django/thunderstore/core/settings.py

```diff
@@ -110,6 +110,8 @@
     CELERY_TASK_ALWAYS_EAGER=(bool, False),
     CELERY_TASK_CREATE_MISSING_QUEUES=(bool, True),
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=(bool, False),
+    CELERY_TASK_SOFT_TIME_LIMIT=(int, 60 * 60),
+    CELERY_TASK_TIME_LIMIT=(int, 60 * 60 + 10),
```

Comment:
~~Soft time limit is one hour and hard limit is one hour and ten seconds? What are we expecting to happen in the last ten seconds?~~ Read [some documentation](https://docs.celeryq.dev/en/latest/userguide/configuration.html#task-soft-time-limit). Any plans to add handling for `SoftTimeLimitExceeded` exceptions in the future?

Also zero-impact nitpick, but why do the two values have to be in different order in the two places where they're used? :angry_laugh:

Reply:
> Any plans to add handling for SoftTimeLimitExceeded exceptions in the future?

Nope, but it's nice to know if it happens via error logs as opposed to just silently failing.

> Also zero-impact nitpick, but why do the two values have to be in different order in the two places where they're used?

Unpredictability is exciting! (I did notice the same after PRing but didn't bother fixing it)